components/segment-bridge/staging/stone-stg-host/kustomization.yaml

```diff
@@ -1,5 +1,4 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-- ../base
-
+  - ../../rbac
```

Comment:
Similarly to the other comment "rbac" isn't really telling me its setting up a service account and associated roles for reading `Usersignup` resources from the `toolchain-host-operator` namespace on the host cluster.

Reply:
The files are being moved around, not modified, to keep this change as small and atomic as possible so to fix the bug. I agree namings can be better, a separate ticket should be opened for it.